testing/benches/all.rs

```diff
@@ -69,3 +70,17 @@ struct Team {
     name: String,
     score: u8,
 }
+
+fn escaping(b: &mut criterion::Bencher) {
+    let string_foo = "foo".to_string();
+    let string_full_escape = "<>&\"\\/";
+    let string_mix_escape = "<foo>bar&foo\"bar\\foo/bar";
```

Comment:
In terms of benchmark cases, I think we roughly need the following to adequately cover the usage surface of this function:

- Short string vs longer string (like some chunk of "lorem ipsum")
- No escapable characters vs ~5% escapable characters

The (almost) everything case (as in `string_full_escape` here) seems not so important to the performance profile of this function, in my view.

How does that sound?

Reply:
Sound goods